figures/serializers.py

```diff
@@ -335,6 +349,7 @@ def get_learners_enrolled(self, course_overview):
         linked
         """
         return get_course_history_metric(
+            site=self.site,
```

Comment:
Why the hack above for `to_representation`, why not do the below instead:

```suggestion
            site=figures.sites.get_site_for_course(course_overview),
```